src/js/views/maps/LayerCategoryListView.js

```diff
@@ -42,36 +48,41 @@ define([
 
       /**
        * The array of layer categories to display in the list
-       * @type {LayerCategoryItemView[]}
+       * @type {ExpansionPanel[]}
        */
-      layerCategoryItemViews: undefined,
+      panels: undefined,
 
       /**
        * Executed when a new LayerCategoryListView is created
-       * @param {Object} options - A literal object with options to pass to the view
+       * @param {object} options - A literal object with options to pass to the view
        */
       initialize(options) {
         if (options.collection instanceof AssetCategories) {
-          this.layerCategoryItemViews = options.collection.map(
-            (categoryModel) => {
-              return new LayerCategoryItemView({ model: categoryModel });
-            },
-          );
+          this.panels = options.collection.map((categoryModel) => {
+            const icon = categoryModel.get("icon");
+            return new ExpansionPanelView({
+              contentViewInstance: new LayerListView({
+                collection: categoryModel.get("mapAssets"),
+                isCategorized: true,
+              }),
+              icon,
+              isSvgIcon: IconUtilities.isSVG(icon),
+              panelsModel: new ExpansionPanelsModel({ isMulti: true }),
+              title: categoryModel.get("label"),
+            });
+          });
```

Comment:
could we just save `options.collection` to the view then instantiate the `ExpansionPanelView`s in `render`? I ask because it would be a more typical pattern (unusual to instantiate sub views in `initialize`) and also avoids iterating through the collection/panels twice.

Reply:
Done